### PR TITLE
Design api integration

### PR DESCRIPTION
**Pull request by Shreya-Kanani**

### CODE REVIEW

File: src/app/core/auth/components/login/login.component.ts [hunk 1]
```diff
@@ -16,10 +16,7 @@ import { Navigation } from 'src/app/shared/common/enum';
   styleUrls: ['./login.component.scss'],
 })
 export class LoginComponent {
-
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/auth/components/login/login.component.ts:19]: create form in oninit not in constructor.

File: src/app/core/components/header/header.component.html [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
-              <li class="spacer"></li>
-              <li *ngIf="isAuthenticated"><a (click)="logout()" title="click here to log out">Logout</a></li>
-          </ul>
-      </div>
-  </div>
-</div>
\ No newline at end of file
+<header>
+  <mat-toolbar color="primary">
+    <div class="header-left">
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/components/header/header.component.html:3]: no need of herf, so remove it

File: src/app/core/components/header/header.component.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter, HostListener } from '@angular/core';
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/components/header/header.component.ts:1]: declare variable before constructor

File: src/app/modules/masters/technology/components/add-technology/add-technology.component.html
Comment (ShubhamPadiyaTatvasoft) [inline src/app/modules/masters/technology/components/add-technology/add-technology.component.html:1]: remove components

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

File: src/app/core/auth/services/login.service.ts [hunk 4]
```diff
@@ -24,29 +30,51 @@ export class LoginService {
...
-  //     }),
-  //   );
-  // }
+  login(payload: LoginModel): Observable<ResponseModel<string>> {
+    return this.http
+    .post<ResponseModel<string>>(`${environment.baseURL}UserAuthentication/Login`, payload)
+    .pipe(
+      map((res: any) => {
+        if (res.result) {
+          console.log(res.result)
```
Comment (Trusha-Tatvasoft) [inline src/app/core/auth/services/login.service.ts:69]: remove console from everywhere

**Review by Trusha-Tatvasoft — APPROVED** [review]

**Pull request merged**